src/main/java/com/derongan/minecraft/deeperworld/config/DeeperConfig.kt

```diff
@@ -20,7 +20,7 @@ object DeeperConfig : IdofrontConfig<DeeperConfig.Data>(deeperWorld, Data.serial
         val remountPacketDelay: TimeSpan = 40.ticks,
         val fall: FallDamageConfig = FallDamageConfig(),
         val time: TimeConfig = TimeConfig(),
-    )
+    ) { val worlds = sections.map { it.world }.toSet()}
```

Comment:
Sorry for being nitpicky but the class body should be on a newline. I should have linked this earlier, but coding conventions explanation is here: https://wiki.mineinabyss.com/contribute/code.html

I'd recommend starting to spam that `Ctrl + Alt + L` shortcut :p

In general if there are no changes made by the autoformatter, that fits the code style guidelines.